### ISSUE TITLE
linuxPackages_latest.facetimehd doesn't build

### ISSUE THREAD

**Issue by yegortimoshenko** [NixOS/nixpkgs#33220]
## Issue description

```
 92 31.0M   92 28.7M    0     0  7353k      0  0:00:04  0:00:04 --:--:-- 6638k/nix/store/h1vzl6bq4wif3m8dd1bw2p3fv4shjg3n-linux-4.14.9-dev/lib/modules/4.14.9/source/Makefile:939: *** "Cannot generate ORC metadata for CONFIG_UNWINDER_ORC=y, please install libelf-dev, libelf-devel or elfutils-libelf-devel".  Stop.
make[2]: *** [Makefile:146: sub-make] Error 2
make[1]: *** [Makefile:24: __sub-make] Error 2
make[1]: Leaving directory '/nix/store/h1vzl6bq4wif3m8dd1bw2p3fv4shjg3n-linux-4.14.9-dev/lib/modules/4.14.9/build'
make: *** [Makefile:9: all] Error 2
'/nix/store/2g5bbhgf7nj0dhgkjq5vdyrvdiahbzwa-busybox-1.27.2/bin/ionice' -> '/nix/store/05w5xdpnirq8yzwzxdi2sfjw6nih6f57-extra-utils/bin/ionice'
'/nix/store/2g5bbhgf7nj0dhgkjq5vdyrvdiahbzwa-busybox-1.27.2/bin/iostat' -> '/nix/store/05w5xdpnirq8yzwzxdi2sfjw6nih6f57-extra-utils/bin/iostat'
builder for ‘/nix/store/03g3sv4f0xiwfp9644x6ac6vcjzhykb8-facetimehd-unstable-2016-10-09-4.14.9.drv’ failed with exit code 2
fetching path ‘/nix/store/z6l3ck1ss258xl1lfyv1ikps3bwqjnki-nixos-prepare-root’...
cannot build derivation ‘/nix/store/pkjp002mn0q6jnnnbh5rl3pfqwpn9nlz-kernel-modules.drv’: 1 dependencies couldn't be built
killing process 2544
```

### Steps to reproduce

```
nix-build path/to/nixpkgs/ -A linuxPackages_latest.facetimehd
```

## Technical details

Checked against 122720ff2bb5cedfbe4820e12c16f1034893abaa.

**Comment by dezgeg** [comment]
Same solution as 5997b4f3db89b738c498bb7786f2e95b3b9b5dcd

**Comment by yegortimoshenko** [comment]
Thanks, @dezgeg!

**Comment by yegortimoshenko** [comment]
Closed via a8d515add59ba9730209cc401a69b07d731de9ae.
